client.js

```diff
@@ -66,8 +66,15 @@ var keycode_to_input = {
   87: 'forward',
 };
 
+var last_update = 0;
```

Comment:
can you name this something more descriptive?

last_input_send_timestamp or something?